gcapi/models.py

```diff
@@ -0,0 +1,2129 @@
+# generated by datamodel-codegen:
```

Comment:
Auto-generated by `generate_models.py`, doesn't need to be reviewed thoroughly, maybe just the overall structure of using pydantic dataclasses.